parameters.yaml

```diff
@@ -31,6 +31,14 @@ account_id:
   required: true
   schema:
     type: string
+data_type:
```

Comment:
I suggest renaming this to `custom_metadata_data_type` or `custom_metadata_type` to avoid confusion